R/combinations-calculate_matrix_metric.R

```diff
@@ -96,13 +96,13 @@ calculate_Bliss <- function(sa1, series_id1, sa2, series_id2, metric) {
   data.table::setnames(sa2, "x", "metric2", skip_absent = TRUE)
 
   # TODO: ensure they're unique?
-  u <- expand.grid(sa1[[series_id1]], sa2[[series_id2]])
+  u <- data.table::CJ(sa1[[series_id1]], sa2[[series_id2]])
```

Comment:
Important question - this functions are giving object in different order - see:

```
expand.grid(sa1[[series_id1]], sa2[[series_id2]]) |> head()
  Var1 Var2
1    1    1
2    2    1
3    3    1
4    4    1
5    5    1
6    6    1
data.table::CJ(sa1[[series_id1]], sa2[[series_id2]]) |> head()
   V1 V2
1:  1  1
2:  1  2
3:  1  3
4:  1  4
5:  1  5
6:  1  6
```
Tests did not failed, but it is for sure that it will not cause any hidden bug?

Reply:
The order does not matter

Reply:
✅